packages/wdio-browserstack-service/src/util.ts

```diff
@@ -17,10 +18,18 @@ import type { UserConfig, UploadType, LaunchResponse, BrowserstackConfig } from
 import type { ITestCaseHookParameter } from './cucumber-types.js'
 import { BROWSER_DESCRIPTION, DATA_ENDPOINT, DATA_EVENT_ENDPOINT, DATA_SCREENSHOT_ENDPOINT } from './constants.js'
 import RequestQueueHandler from './request-handler.js'
+import * as util from 'node:util'
```

Comment:
Let's move this to the top next to the other node imports
```suggestion
import util from 'node:util'
```

Reply:
done https://github.com/webdriverio/webdriverio/pull/10183/commits/e8539f4a450860e198b32d961dc13d55fdfb7658